packages.json

```diff
@@ -1518,6 +1518,7 @@
   "https://github.com/Moya/Moya.git",
   "https://github.com/mpangburn/FunctionKit.git",
   "https://github.com/mpapp/mptimer.git",
+  "https://github.com/mrackwitz/Version.git",
```

Comment:
Is this an intentional part of this PR? It's fine to add a library, but I was surprised to see it here.

Reply:
Or is this test data?

Reply:
Both? 😄 

I included it in the MR so that we could see how CI handles a new library being added (so it actually runs the full length of the new script!)

We can, of course, remove it before merging (or just leave it as it's one missing library)

Reply:
It makes good test data, let's leave it!